app/src/main/res/layout/activity_browser.xml

```diff
@@ -141,6 +141,13 @@
 
     </android.support.design.widget.CoordinatorLayout>
 
+    <include
+        layout="@layout/include_find_in_page"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:visibility="gone"
+        tools:visibility="visible" />
```

Comment:
Can we remove the `tools:visibility` here so that we can see the more common search use case in the xml?